### DISCUSSION TITLE
Welcome to DevXan Discussions!

### DISCUSSION THREAD

**Discussion by devxan** [devxan/devxan#1] [General]
## 👋 Welcome!
If you want to talk to me through Github (or you have a question about my awesome profile readme), this is the place to do it!

- 🧠 Just be mindful that I am a human and that everyone can see what you share here, so maybe check out the [Github Community Guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines)
- 💡 If you want to share an idea or change for the profile README.md, please do so with Github [Issues](https://github.com/devxan/devxan/issues)

**Comment by ghost** [comment]
Wow, I didn't think of using discussions for general chatting (not related to a project)!

**Reply by devxan** [reply]
:O the first person to actually use DevXan Discussions! :o 🎉

**Reply by Wjplatformer** [reply]
# Woah!
I did not know there was discussions! I thought there was only issues! 😁

**Comment by Wjplatformer** [comment]
# Your Archive is still working though...lol
Xan! Your -Xanimation- account on scratch has a 109 followers at the moment, and your archived markdown shows a 110! Magical! 😮

**Comment by Wjplatformer** [comment]
# Invite for WJPScript
Hi Xan! I have invited you to work on my new project which is a private repo. I am using JSON for information transfer and just wanted to know if you would accept this invite. Don't worry, I can always remove you if you do not want to be a collaborator :grin: (like if you don't accept this invite too lol). Oh yes, if you want to change to a more universal name, then well okay :smile:

**Reply by Wjplatformer** [reply]
So, are you not free to work on my project? No worries. 😅

**Reply by devxan** [reply]
Yeah, sorry but no thanks @Wjplatformer.

**Reply by Wjplatformer** [reply]
Oh, no worries! Thanks for your time!

**Reply by devxan** [reply]
> Oh, no worries! Thanks for your time!

You're welcome! Thanks for being so understanding 🙂

**Comment by Wjplatformer** [comment]
# Some usage of your stuff...
Sorry Xan for this late message when it is already on my README! I have been using your gif wave via link. If you want me to take it down, then sure. So this means that I am asking for your permission. sorry.
##### Also, I was wondering how to include badges with some statistics in markdown (sorry if it's vague).

**Reply by devxan** [reply]
So, that's perfectly fine. To be honest, I found the waving hand gif in a random readme as well. I would recommend downloading it and adding it into your profile files though (in a folder marked "assets" or something similar). 

I get all my badges from https://shields.io/. They already have some readily made badge templates but you can also ask around on the internet for someone to import your data into a shield badge without learning how to do so yourself. 
I use the style "For the badge" with a custom color on all of mine. (TODO: change it)

**Reply by Wjplatformer** [reply]
Thanks!

**Reply by devxan** [reply]
You're welcome! By the way, your waving hand image is not functional at the moment.

**Reply by Wjplatformer** [reply]
Well... it is functional. But to be safe, I'll download the gif :)

**Comment by Wjplatformer** [comment]
# Hello Xan, I would like to open a pull request with a badge on the users online for ScratchxDiscord organization. 
I just need the server ID for the badge. I feel like upgrading the website bit by bit as it feels too empty.

**Reply by Wjplatformer** [reply]
You may add that by yourself btw. I just wanna try and open my first PR >:)

**Reply by devxan** [reply]
You can always join [our Discord server](https://discord.gg/bUtXEtHKKC)! Here's its ID anyways: 805881494107193354

**Reply by Wjplatformer** [reply]
thanks :smile:

**Comment by Wjplatformer** [comment]
![image](https://user-images.githubusercontent.com/82447031/159125593-51186383-91f3-4a4e-a11d-57acbeed167b.png)
Some website improvements :). There are links and everything, so I will pull request you soon. Do you accept this?

**Reply by devxan** [reply]
Maybe! (In the future, you can make your own discussions instead of commenting on this discussion to keep them organized.) 
I will probably edit it but it looks great from your image.

**Reply by Wjplatformer** [reply]
Thanks! Sorry, But maybe a few days later if you want me to PR. :hugs:

**Reply by Wjplatformer** [reply]
Me done qwp

**Comment by Wjplatformer** [comment]
Your wave.gif has enlarged sir

**Reply by devxan** [reply]
Thanks for the info! Looks like GitHub updated something! 🤔 Any idea how I could fix it, @Wjplatformer?

**Reply by Wjplatformer** [reply]
aw, doesn't seem to work, even uploading it again! (lol). I would try to find some way to solve this issue, might even have to upload another gif that has a smaller size (file size), but no promises!

**Reply by Wjplatformer** [reply]
**Furiously typing* * SIR I have found a solution Very simple for the width set it to 5% trust me it is fixed on my readme<br> `<img src="https://raw.githubusercontent.com/Wjplatformer/Wjplatformer/main/wave.gif" width="5%">`

**Reply by devxan** [reply]
Thank you, @Wjplatformer!

**Comment by Wjplatformer** [comment]
Right now I don't have my laptop with me, so I don't have my code editor to test things out. But the most obvious solution that I can think of is to change the width of the `<image>` tag to around 10px instead of 30px.

**Comment by Wjplatformer** [comment]
Hello kind sir, I am here to ask you about something to do with my website. See, I have been working on it [here](https://tel-code.github.io). And I have not committed this new change. Here is the picture: 
![sos small space](https://user-images.githubusercontent.com/82447031/172033318-9f1c3cd1-b31a-4b62-bcab-f90cd4912294.png)
Small space between a `<div>` element and the html page itself. Any idea on how to remove it? Here is my css code: <br>
`div{
    box-sizing: border-box;
    width: 100%;
    height: 1000px;
    display: inline-block;
    background: rgb(176, 202, 192);
    margin-top: 1000px;
    padding: 0; 
    box-sizing:border-box; 
}`<br>
alongside with some html, if that is the problem <br>
         `<body>
          <nav>
            <ul>
              <li><a href="index.html">Home</a></li>
              <li><a href="projects.html">Projects</a></li>
              <li><a href="services.html">Services</a></li>
              <li><a href="timings.html">Timings</a></li>
            </ul>
          </nav>
          <h1 style="font-size: 80px; margin: 1em; font-family: 'Trebuchet MS', Arial;">print('hello world') to coding</h1>
          ***<div></div>***
        </body>`
##### apologies for the cramming, I have no idea how to make them a list

**Reply by devxan** [reply]
Ummm idk. Use stack overflow and join this discord server I'm in for developers: https://citrus.lol/invite

**Reply by Wjplatformer** [reply]
Well, in 11 months time :)

**Reply by Wjplatformer** [reply]
anyways I have fixed it :)

**Comment by Wjplatformer** [comment]
Hey there Xan, I have something important related to scratch that affects the safety of users <br>
In which @griffpatch has created tutorials on how to do image scanning in under 8 seconds. <br>
Which means, that you can scan your own face, and data can be sent to cloud variables. <br>
The face can then be drawn to other's screen. So <br>
***dangerous people***<br>
may create such projects to capture each other's faces or even... you know.<br>
So I feel that we should alert the scratch team in some way :|<br>
##### respond or else

**Reply by devxan** [reply]
People have already done this. A person I've friended on discord sent me a project like this on TurboWarp and sent me screenshots of my finger covering the camera.

**Reply by Wjplatformer** [reply]
Oh no...

**Comment by Wjplatformer** [comment]
#### Pull Shark achievement for devxan 🥳

**Comment by Wjplatformer** [comment]
Using github mobile >:)

**Reply by Wjplatformer** [reply]
There's sesame seeds in my mouth and I hate it because it is so bad.

**Reply by devxan** [reply]
Interesting 🤨

**Reply by Wjplatformer** [reply]
I was drinking something which had sesame seeds in it which I did not expect at all

**Reply by Wjplatformer** [reply]
Something very sweet. And I thought the sesame seeds were chocolate. It was a gift, I did not know what was inside 😅

**Reply by devxan** [reply]
### 😮🧐

**Reply by Wjplatformer** [reply]
I'm returning from my trip soon and can have my laptop back :)

**Reply by devxan** [reply]
Cool!

**Comment by Wjplatformer** [comment]
Hellow, just wanted to ask you Smth about my website. You would receive an email and perhaps I can give you the link

**Reply by Wjplatformer** [reply]
I was pretty sudden don't you think?

**Reply by devxan** [reply]
Yeah. I thought I was replying to something on one of my own repos - oops!

**Comment by Wjplatformer** [comment]
I'm heading back to my laptop tmr. I would then be able to do a lot of coding at once :)

**Reply by devxan** [reply]
Cool! 👌

**Reply by Wjplatformer** [reply]
After a 6 hour journey across a country I am officially 1h away from my home :)

**Reply by Wjplatformer** [reply]
And my phone won't be with me anymore :(

**Reply by devxan** [reply]
Wow! But why's that? 🤨 (Also did your send me that sketchy looking email text?)

**Reply by Wjplatformer** [reply]
Erm, this is actually not my phone so I have to return it but I did not remember sending you an email, never have I ever. But I am home now in sunny Singapore :)

**Comment by Wjplatformer** [comment]
xan... I might have made a mistake 😬<br>
![image](https://user-images.githubusercontent.com/82447031/174288802-d50d942a-15dc-4345-a48b-a165d50cf972.png)
![thetruthxan](https://user-images.githubusercontent.com/82447031/174288828-13652d22-acfc-4980-ba48-5db4f5ec90ee.png)
or is that true xan?

**Reply by devxan** [reply]
My scratch account -Xanimation- was not banned.

**Reply by Wjplatformer** [reply]
well might be me then...<br>
I will message them sowwy

**Reply by Wjplatformer** [reply]
Also do note that your ukraine project was taken down :( (again)

**Reply by Wjplatformer** [reply]
okie, this asian gonna go offline to rest 😴 @devxan , see you tmr (if you're not busy then)

**Reply by devxan** [reply]
Okay. (The Scratch Team took down my project again) 💀

**Reply by Wjplatformer** [reply]
done resting when your time is 9.23p.m.

**Comment by Wjplatformer** [comment]
Good morning (night) fellow developers :)

**Reply by devxan** [reply]
Hello! 👋

**Reply by Wjplatformer** [reply]
69 subscribers. I don't know what that means but it is cool.

**Reply by devxan** [reply]
😄 I don't know exactly what it means, but _the number relates to sexual intercourse_ 😶. My YouTube channel now has 70 subscribers 👀!

**Reply by Wjplatformer** [reply]
erm. Okie, I knew it all along. Heh. *slowly runs away*

**Reply by Wjplatformer** [reply]
*relates 😶*

**Reply by Wjplatformer** [reply]
yes. Really 😶. But we don't talk about that here, do we? EDIT: I CREATED A NEW DISCUSSION

**Reply by devxan** [reply]
> Slowly runs away

*__LMAO__* (the irony tho)

> But we don't talk about that here, do we?

We could. I have no reason not to in this publicly viewable space (😶)

**Reply by devxan** [reply]
Also you should stop talking on this discussion. 
![Screenshot_20220621-203400](https://user-images.githubusercontent.com/57809064/174937999-815e6f1b-24f6-41ac-a300-87be177eae4d.png)
We're almost to 69 comments 🗿 so please make a new discussion lol. I want to keep it at the funni number

Edit: 
![Screenshot_20220621-203552](https://user-images.githubusercontent.com/57809064/174938179-fc80680c-c09f-4092-ac37-1bdbb4dd7f46.png)
This was the 69th comment GG WE DID IT BOIS
